torch/fx/node.py

```diff
@@ -109,11 +109,11 @@ class Node:
     - ``output`` contains the output of the traced function in its ``args[0]`` attribute. This corresponds to the "return" statement
       in the Graph printout.
     """
-    def __init__(self, graph: 'Graph', name: str, op: str, target: 'Target',
+    def __init__(self, graph: 'Graph', naming_fn: Callable, op: str, target: 'Target',
```

Comment:
This is BC-breaking. Is this really necessary?

Reply:
not strictly necessary, I can remove at the cost of minor ugliness in `_Namespace`, which seems like reasonable thing to do.